packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureButton.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps, BoxStyleProps} from '@twilio-paste/box';
+import {DisclosurePrimitive} from '@twilio-paste/disclosure-primitive';
+import {ChevronDisclosureExpandedIcon} from '@twilio-paste/icons/esm/ChevronDisclosureExpandedIcon';
+
+import {SidebarDisclosureContext} from './SidebarDisclosureContext';
+
+export interface SidebarDisclosureButtonProps extends BoxProps {
+  onClick?: () => void;
+  level: 0 | 1;
+}
+
+const levelPaddingMap: Partial<
+  {
+    [key in SidebarDisclosureButtonProps['level']]: Partial<BoxStyleProps>;
+  }
+> = {
+  0: {
+    paddingY: 'space40',
+    paddingLeft: 'space40',
+    paddingRight: 'space40',
```

Comment:
These three lines can just be `padding: 'space40'` no?

Reply:
oof - just updated.